src/contexts/OntologyStyles.tsx

```diff
@@ -51,9 +56,14 @@ type OntologyStylesProviderProps = {
   children: React.ReactNode;
 }>;
 
-const OntologyStylesContext = createContext<OntologyStylesContextProps | null>(null);
+const OntologyStylesContext = createContext<OntologyStylesContextProps | null>(
+  null
+);
 
-const OntologyStylesProvider: React.FC<OntologyStylesProviderProps> = ({ service, children }) => {
+const OntologyStylesProvider: React.FC<OntologyStylesProviderProps> = ({
```

Comment:
FYI
VSCode's "Save without formatting" can be handy to control unrelated whitespace changes in PRs